demonstrations/tutorial_here_comes_the_sun.py

```diff
@@ -0,0 +1,570 @@
+    \{\Omega \in \mathbb{C}^{N\times N}: \Omega^\dagger=-\Omega, \operatorname{Tr}[\Omega]=0\}.
+
+The first condition means that the elements :math:`\Omega` are *skew-Hermitian* and the
+second condition means that their trace vanishes.
+We will use so-called canonical coordinates for the algebra which are simply the coefficients
+in the Pauli basis. That is, we consider the Pauli basis elements multiplied with the
+imaginary unit :math:`i`, except for the identity:
+
+.. math::
+
+    G_m \in \mathcal{P}^{(n)} = i \left\{I,X,Y,Z\right\}^n \setminus \{i I^n\}.
+
+A Lie algebra element :math:`\Omega` can be written as
+
+.. math::
+
+    \Omega = \sum_{m=1}^d \theta_m G_m,\quad \theta_m \in \mathbb{R}
+
+and those coefficients :math:`\theta` are precisely the canonical coordinates.
+You may ask why we included the prefactor :math:`i` in the :math:`G_m`, and why we excluded
+the identity (times :math:`i`). This is to match the properties of :math:`\mathfrak{su}(N)`:
+The prefactor makes the basis elements skew-Hermitian and the identity would not have a
+vanishing trace. Indeed, one can check that the dimension of :math:`\mathfrak{su}(N)` is
+:math:`4^n-1` and that there are :math:`4^n` Pauli words, so that one Pauli word had to go
+in any case... We can use the canonical coordinates of the algebra to express a *group element* in
+:math:`\mathrm{SU}(N)` as well, and the ``qml.SpecialUnitary`` gate we will use is defined as
+
+.. math::
+
+    U(\bm{\theta}) = \exp\left\{\sum_{m=1}^d \theta_m G_m \right\}.
+
+The number of coordinates and of Pauli words in :math:`\mathcal{P}^{(n)}` is :math:`d=4^n-1`.
+Therefore, this will be the number of parameters a single ``qml.SpecialUnitary`` gate acting on
+:math:`n` qubits will take. For example, it takes just three parameters for a single qubit, which
+is why :class:`~pennylane.Rot` and :class:`~pennylane.U3` take three parameters and may
+produce *any* single-qubit rotation. It takes a moderate number of 15 parameters for two qubits,
+but already requires 63 parameters for three qubits.
+
+Obtaining the gradient
+----------------------
+
+In variational quantum algorithms, we typically use the circuit to prepare a quantum state and
+then we measure some observable :math:`H`. The resulting real-valued output is considered to be the
+cost function :math:`C` that should be minimized. If we want to use gradient-based optimization for
+this task, we need a method to compute the gradient :math:`\nabla C` in addition to the cost
+function itself. As derived in the publication [#wiersema]_, this is possible on quantum hardware
+for :math:`\mathrm{SU}(N)` gates as long as the gates themselves can be implemented.
+The implementation in PennyLane follows the decomposition idea described in App. F3, the
+main text of [#wiersema]_ additionally proposes a method that scales better in some scenarios
+but requires additional gates to be available on the quantum hardware. Here we will focus on the
+former method.

```

Comment:
I think this reads better

```suggestion
The implementation in PennyLane follows the decomposition idea described in App. F3, but the
main text of [#wiersema]_ proposes an additional method that scales better in some scenarios
(the caveat being that this method requires additional gates to be available on the quantum hardware).
Here, we will focus on the former method.
```